src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -72,6 +73,50 @@ class TextGenerationInput(BaseModel):
         "can compute metric in a batched fashion. ",
     )
 
+    @validator("session_ids")
+    def validate_session_ids(cls, value, values) -> Union[None, List[str]]:
```

Comment:
let's move this to a helper function in a separate file

Reply:
yeah, good idea.